infra/resources/modules/commons/function_async.tf

```diff
@@ -16,10 +16,14 @@ locals {
     FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
     FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
-    COSMOSDB_ENDPOINT      = module.cosmosdb_account.endpoint
-    COSMOSDB_DATABASE_NAME = module.cosmosdb_sql_database_trial.name
-    EVENTHUB_NAMESPACE     = "${module.event_hub.name}.servicebus.windows.net"
-    SERVICEBUS_NAMESPACE   = "${local.servicebus_namespace}.servicebus.windows.net"
+    COSMOSDB_ENDPOINT                 = module.cosmosdb_account.endpoint
+    COSMOSDB_DATABASE_NAME            = module.cosmosdb_sql_database_trial.name
+    EVENTHUB_NAMESPACE                = module.event_hub.name
+    SERVICEBUS_NAMESPACE              = local.servicebus_namespace
```

Comment:
**note**: this should work even without the `servicebus.windows.net` suffix

Reply:
In the end, I decided not to revert this change, because it looks like the `<yournamespace>.servicebus.windows.net` is the fullyQualifiedNamespace which includes the `namespace`.

So, in order to let things working properly, I updated the creation of the clients that needed the `fullyQualifiedNamespace` adding the suffix.